### ISSUE TITLE
tensorflow_util_test fails

### ISSUE THREAD

**Issue by pervazea** [cmu-db/peloton#1216]
On a debug build, Ubuntu Xenial (16.04), current master.

Preferably fix so the test passes.

If this is not possible (e.g. if we are not ready for this test to be run yet), disable it for now.

```
UpdateCTestConfiguration  from :/home/pakhtar/proj/master/peloton/build/DartConfiguration.tcl
Parse Config file:/home/pakhtar/proj/master/peloton/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/pakhtar/proj/master/peloton/build/DartConfiguration.tcl
Parse Config file:/home/pakhtar/proj/master/peloton/build/DartConfiguration.tcl
Test project /home/pakhtar/proj/master/peloton/build
Constructing a list of tests
Done constructing a list of tests
Checking test dependency graph...
Checking test dependency graph end
test 24
    Start 24: tensorflow_util_test

24: Test command: /home/pakhtar/proj/master/peloton/build/test/tensorflow_util_test "--gtest_color=yes" "--gtest_output=xml:/home/pakhtar/proj/master/peloton/build/test/tensorflow_util_test.xml"
24: Test timeout computed to be: 1500
1/1 Test #24: tensorflow_util_test .............***Exception: SegFault  0.12 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.12 sec

The following tests FAILED:
	 24 - tensorflow_util_test (SEGFAULT)
Errors while running CTest
```

**Comment by malin1993ml** [comment]
I ran the test on my own Ubuntu 16 machine and got the same error:
```
(gdb) run
Starting program: /home/lin/peloton/build/test/tensorflow_util_test 
y[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff062271f in _GLOBAL__sub_I_mathlimits.cc () from /usr/local/lib/libtensorflow_framework.so
(gdb) 
```

**Comment by saatviks** [comment]
Upgrading Tensorflow version to 1.5 solves the problem on 16.04. Will test on 14.04 and submit a PR.

**Comment by malin1993ml** [comment]
Fixed in #1219.
